acceptance-tests/tests/common.go

```diff
@@ -0,0 +1,355 @@
+	BeTrue               = gomega.BeTrue
+	BeFalse              = gomega.BeFalse
+	HaveOccurred         = gomega.HaveOccurred
+	Succeed              = gomega.Succeed
+	MatchError           = gomega.MatchError
+	BeClosed             = gomega.BeClosed
+	Receive              = gomega.Receive
+	BeSent               = gomega.BeSent
+	MatchRegexp          = gomega.MatchRegexp
+	ContainSubstring     = gomega.ContainSubstring
+	HavePrefix           = gomega.HavePrefix
+	HaveSuffix           = gomega.HaveSuffix
+	MatchJSON            = gomega.MatchJSON
+	MatchXML             = gomega.MatchXML
+	MatchYAML            = gomega.MatchYAML
+	BeEmpty              = gomega.BeEmpty
+	HaveLen              = gomega.HaveLen
+	HaveCap              = gomega.HaveCap
+	BeZero               = gomega.BeZero
+	ContainElement       = gomega.ContainElement
+	BeElementOf          = gomega.BeElementOf
+	ConsistOf            = gomega.ConsistOf
+	ContainElements      = gomega.ContainElements
+	HaveKey              = gomega.HaveKey
+	HaveKeyWithValue     = gomega.HaveKeyWithValue
+	BeNumerically        = gomega.BeNumerically
+	BeTemporally         = gomega.BeTemporally
+	BeAssignableToTypeOf = gomega.BeAssignableToTypeOf
+	Panic                = gomega.Panic
+	PanicWith            = gomega.PanicWith
+	BeAnExistingFile     = gomega.BeAnExistingFile
+	BeARegularFile       = gomega.BeARegularFile
+	BeADirectory         = gomega.BeADirectory
+	HaveHTTPStatus       = gomega.HaveHTTPStatus
+	And                  = gomega.And
+	SatisfyAll           = gomega.SatisfyAll
+	Or                   = gomega.Or
+	SatisfyAny           = gomega.SatisfyAny
+	Not                  = gomega.Not
+	WithTransform        = gomega.WithTransform
+)
+
+// Path and all the other global vars
+var (
+	Path        string = "/data"
+	pwd         string
+	err         error
+	manifests   string = "/manifests"
+	file        string = "/package.json"
+	target      string = "/package.json"
+	GoMainFile  string = "/main.go.template"
+	GoMainFileT string = "/main.go"
+)
+
+// getCRDAcmd returns the command according to os
+func getCRDAcmd() string {
+	if runtime.GOOS == "darwin" || runtime.GOOS == "linux" {
+		return "./crda"
+	} else if runtime.GOOS == "windows" {
+		return "./crda.exe"
+	}
+	return "crda"
+}
+
+// GetAbsPath returns absolute path
+func GetAbsPath() {
+	pwd, err = helper.Getabspath(Path)
+	if err != nil {
+		acclog.ErrorLogger.Println(err)
+	}
+}
+
+// InitVirtualEnv makes a new virtual env for python
+func InitVirtualEnv() {
+	cmd := exec.Command("/bin/sh", "-c", "cd "+pwd+"; python3 -m venv env; source env/bin/activate; pip3 install -r requirements.txt;")
+	stdout, err := cmd.Output()
+	acclog.InfoLogger.Println(string(stdout))
+	if err != nil {
+		acclog.ErrorLogger.Println(err)
+	}
+}
+
+// InstallNpmDeps runs npm install
+func InstallNpmDeps() {
+	cmd := exec.Command("npm", "install")
+	cmd.Dir = "data"
+	stdout, err := cmd.Output()
+	acclog.InfoLogger.Println(string(stdout))
```

Comment:
every log is in stdout, You can set this logging globally to `stdout` then use that in your codebase

Reply:
If you can, Try using the already implemented logging lib "github.com/rs/zerolog/log"

Reply:
okay